<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInteractorStyleVesselSegmentation2</class>
 <widget class="QWidget" name="QInteractorStyleVesselSegmentation2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QInteractorStyleVesselSegmentation2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QLabel" name="label_21">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000a5;&quot;&gt;Current Stage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Vessel segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QRadioButton" name="smoothCurveRadioButton">
          <property name="text">
           <string>Smooth Curve</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="polygonRadionButton">
          <property name="text">
           <string>Polygon</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonMagnetic">
          <property name="text">
           <string>Magnetic</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="fillContourPushButton">
          <property name="text">
           <string>Fill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanOnePushButton">
          <property name="text">
           <string>Clean One</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanAllPushButton">
          <property name="text">
           <string>Clean all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Keyboard key:
Delete: 
Deleting focusing node. 
Shift + Delete: 
Deleting focusing focusing contours.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="pushButtonFillSlices">
          <property name="text">
           <string>Fill Slices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxFillSlicesBegin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>To:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxFillSliceEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(226, 226, 226)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="activeLabelLabel_3">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Active Label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>No label</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/transparentbox.png</normaloff>:/icons/transparentbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vessel Wall</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lumen</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Calcification</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/greenbox.png</normaloff>:/icons/greenbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hemorrhage</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/yellowbox.png</normaloff>:/icons/yellowbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LRNC</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/lightbluebox.png</normaloff>:/icons/lightbluebox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LM</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/purplebox.png</normaloff>:/icons/purplebox.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonNormal">
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonLumenSegmentation">
        <property name="text">
         <string>Lumen Segmentation</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonVesselWallSegmentation">
        <property name="text">
         <string>Vessel Wall Segmentation</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetMode">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pageNormal">
         <layout class="QVBoxLayout" name="verticalLayout_7"/>
        </widget>
        <widget class="QWidget" name="pageVesselWallSegmentation">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Auto Vessel Wall Segmentation</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxVesselWallThickness">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="suffix">
               <string> px</string>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAutoVesselWallSegmentation">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="vesselFrame">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(226, 226, 226)</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="activeLabelLabel">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Vessel Wall Label:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="vesselWallComboBox">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>13</width>
                   <height>13</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Vessel Wall</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
                    <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="lumenLabel">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Lumen Label:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="lumenComboBox">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>13</width>
                   <height>13</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Lumen</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
                    <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageLumenSegmentation">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Auto lumen segmentation:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Value</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="autoLumenSegmentationHorizontalSlider">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>60</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="autoLumenSegmentationSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>60</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAutoLumenSegmentation">
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="vesselFrame_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(226, 226, 226)</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="activeLabelLabel_2">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Vessel Wall Label:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="vesselWallComboBox_2">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>13</width>
                   <height>13</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Vessel Wall</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
                    <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="lumenLabel_2">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="text">
                  <string>Lumen Label:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="lumenComboBox_2">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                  </font>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>13</width>
                   <height>13</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Lumen</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
                    <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
     <zorder>radioButtonLumenSegmentation</zorder>
     <zorder>radioButtonVesselWallSegmentation</zorder>
     <zorder>radioButtonNormal</zorder>
     <zorder>stackedWidgetMode</zorder>
     <zorder>frame_6</zorder>
     <zorder>label_2</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>206</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Main/resources/IADE_Analyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>autoLumenSegmentationHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>autoLumenSegmentationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoLumenSegmentationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>autoLumenSegmentationHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
